styleguide/jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  preset: 'ts-jest/presets/js-with-ts',
+  moduleDirectories: ['node_modules', '<rootDir>/src/'],
+  testPathIgnorePatterns: ['<rootDir>/dist/', '<rootDir>/tmp/'],
```

Comment:
When does the `tmp` dir become created? Haven't seen it day-to-day

Reply:
I think it's a carry over from the webpack dev server in OperatorUI. ✂️